### PR TITLE
Added new page for change email - email sent. Added spinner that over…

### PR DESCRIPTION
**Pull request by joseluisgraa**
…writes current B2C spinner. Moved some things to services. Updated post-build script

### CODE REVIEW

File: b2c-app/src/colours.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$ncs-base-color: #207e77;
```
Comment (joseluisgraa) [inline b2c-app/src/colours.scss:1]: Not sure where this file should go in current folder structure. Any ideas?

File: b2c-app/src/components/PageContainer/PageContainer.js [hunk 2]
```diff
@@ -50,15 +50,19 @@ class PageContainer extends React.Component {
...
-            <div className="govuk-width-container">
-                <components.Breadcrumbs />
-                {pageLevelErrorContainer}
-                <main className="govuk-main-wrapper" >
-                    <div className="govuk-grid-row">
-                        {pageColumns}
-                    </div>
-                </main>
+            <div>
+                <components.Spinner showSpinner={this.props.pageConfig.showSpinner} text={this.props.pageConfig.spinnerText} />
```
Comment (joseluisgraa) [inline b2c-app/src/components/PageContainer/PageContainer.js:54]: I've added a spinner, there isn't a govuk one so we might need to tweak it if they don't like it as is. In the spinner class I have also started observing changes in the DOM to detect when B2C shows/hides its own overlay which looks terrible, and then we show our custom one which to me looks nicer

File: b2c-app/src/pages/EmailSent.js [hunk 3]
```diff
@@ -33,6 +86,15 @@ class EmailSent extends React.Component {
                 .
             </ components.Paragraph>
 
+        const resendChangeEmailParagraph =
+            <components.Paragraph>
+                If you don't receive an email after this time you can&nbsp;
+                <a href="/#" className="govuk-link"
```
Comment (joseluisgraa) [inline b2c-app/src/pages/EmailSent.js:92]: I'll see if I can include this as another variation of our Link component

File: b2c-app/src/colours.scss (hunk 1 above)
Comment (DrogoNevets) [reply b2c-app/src/colours.scss:1]: create a sub dir under src
`/src/styles` we then have somewhere for extra styles if we need them

**Pull request merged**